plugins/stack-overflow-backend/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: [require.resolve('@backstage/cli/config/eslint.backend')],
+};
```

Comment:
now that we have package roles, these eslintrc files should always be exactly like this

https://github.com/backstage/backstage/blob/8b026631fd9f794387e12958404ab0ef812c619d/packages/app/.eslintrc.js#L1